ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -427,6 +427,21 @@ tests:
     test:
     - ref: openshift-e2e-test
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-default-config
```

Comment:
Shall we also need to add the job in openshift-installer-release-4.{18,19,20}.yaml ?

Reply:
I am not sure if `backport` PR would also trigger the pre-submit jobs, if yes, the jobs are defined in `openshift-installer-release-4.18.yaml. For earlier version, so far, it only supports 4.18, so we do not need to consider that yet.

@r4f4 what is your option here ?

Reply:
I think we can add the 4.18 job in this PR. In theory we wouldn't need the 4.19 one but I always add just in case.

Reply:
sure, I think I can add the job to openshift-installer-release-4.{18,19,20}.yaml to make sure there's nothing missed.